src/components/content/catalog/services/details/ServiceProvider.tsx

```diff
@@ -238,19 +238,23 @@ function ServiceProvider({
 
     return (
         <>
+            {serviceDetails && unregisterServiceId === serviceDetails[0].id ? unregisterTips : ''}
```

Comment:
we need a better logic to clear unregister tips. 
Best would be to simply load data again and let the useEffect to render the component again.